### PR TITLE
Migration documentation to PromptL

### PR DESCRIPTION
**Pull request by csansoon**
:)

### CODE REVIEW

File: docs/guides/prompt-manager/migrate-to-promptl.mdx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+### Steps
+
+Now, the contents of steps can be defined directly inside.
+
+```markdown
+/* Before: */
+
+This is one step
+<step />
```
Comment (cesr) [inline docs/guides/prompt-manager/migrate-to-promptl.mdx:44]: response

Comment (csansoon) [reply docs/guides/prompt-manager/migrate-to-promptl.mdx:44]: true!

**Review by andresgutgon — APPROVED** [review]
promptL forever 💪

File: docs/guides/prompt-manager/migrate-to-promptl.mdx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+description: Learn how to migrate your prompts to PromptL
+---
+
+## Overview
+
+[PromptL](https://promptl.ai/) is the next step in defining prompts for LLMs. It builds on the strengths of the syntax used in Latitude, addressing user feedback to create a simpler, more intuitive, and standardized language for prompt design.
+
+This guide will walk you through the process of migrating your existing prompts to PromptL, highlighting the key changes and improvements.
+
+> **Why migrate to PromptL?**
```
Comment (geclos) [inline docs/guides/prompt-manager/migrate-to-promptl.mdx:12]: missing `-`?

**Review by geclos — APPROVED** [review]
beautiful

File: docs/guides/prompt-manager/migrate-to-promptl.mdx (hunk 2 above)
Comment (csansoon) [reply docs/guides/prompt-manager/migrate-to-promptl.mdx:12]: No, this is used as a pseudo-title

**Pull request merged**